Shared/Actions/CreateMenuItem.swift

```diff
@@ -0,0 +1,507 @@
+import Foundation
+import AppIntents
+import SwiftUI
+
+// MARK: Action
+struct CreateMenuItem: AppIntent {
+	static let title: LocalizedStringResource = "Create Menu Item"
+
+	static let description = IntentDescription(
+"""
+Create a menu item with a title, subtitle and icon.
+
+You can later use one or more of these Items in a "Choose from List" action.
+
+Add an "Add to Variable" action below this to populate a list and then use that variable in the "Choose From List" action.
+""",
+		categoryName: "Rich Menu",
+		searchKeywords: [
+			"menu",
+			"menu item",
+			"choose from menu",
+			"rich menu"
+		]
+	)
+
+	static var parameterSummary: some ParameterSummary {
+		When(\.$menuTitle, .hasAnyValue) {
```

Comment:
This conditional is unnecessary. It's weird that the options start showing only when you start writing the title.